GRIDAPPSD/GOSS-GridAPPS-D
# Adding optional automatic simulation pause after publish

This adds the option to specify the helicsgossbridge to pause the simulation automatically after publishing measurements.

To test:
Add the key:value pair "pause_after_measurements": true to the "simulation_config" dictionary value in the simulation request. This should cause the simulation to enter the PAUSE state automatically after every time the helicsgossbridge  sends measurements. This should be only be utilized by the Simulation class in gridappsd-python for application testing purposes.
